impeller/entity/entity_pass.cc

```diff
@@ -548,6 +553,11 @@ bool EntityPass::OnRender(
     }
   }
 
+  if (wait_until_completed) {
+    FML_DCHECK(pass_context.IsActive());
```

Comment:
we need to add `SyncMode` when the last `CommandBuffer` is submitted. So we need to make sure `pass_context.IsActive()` is true at this time.